src/components/dataset-query/dataset-query.js

```diff
@@ -4,8 +4,8 @@ import './dataset-query.scss'
 import LoadingElement from '../generic-elements/loading-element';
 
 
-const DatasetQuery = ({ defaultQuery, onQueryDataset, onCancelQuery, queryFailureMessage, isLoading, hasUserSubmittedQuery }) => {
-  const [queryText, setQueryTextRaw] = useState(defaultQuery)
+const DatasetQuery = ({ freestyleQueryText, onQueryDataset, onCancelQuery, queryFailureMessage, isLoading, hasUserSubmittedQuery }) => {
+  const [queryText, setQueryTextRaw] = useState(freestyleQueryText)
```

Comment:
Looks good. Just to make sure I'm on the same page: we're only syncing the query box text on query submission? I'm absolutely fine with that as long as that's what we want (I reckon it would slam redux to have it update the global query text on every keystroke for example)

Reply:
Yep, it just fills the text in after the page re-renders which will only happen on the initial load (with the default query) or if either instance submits a query

Reply:
^ Actually each tab is re-rendered when it is selected right now to prevent a weird bug with plotly. But I am assuming we will want to change that eventually